zds/utils/signals.py

```diff
@@ -1,7 +1,63 @@
 from django.dispatch import Signal
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.apps import apps
+import logging
+import os
+
+# Logger Setup
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.ERROR)
+current_dir = os.path.dirname(os.path.abspath(__file__))
+log_file = os.path.join(current_dir, "spam_signals.log")
+
+# File Handler
+handler = logging.FileHandler(log_file)
+handler.setLevel(logging.ERROR)
+formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
+handler.setFormatter(formatter)
```

Comment:
Dans les fichiers de code du projet, on définit le *logger* avec seulement cette simple ligne : 

```suggestion
logger = logging.getLogger(__name__)
```

Le reste est géré dans la configuration : 

https://github.com/zestedesavoir/zds-site/blob/5b98a51b5aece0acdfa15d65b0efc71577dddba9/zds/settings/abstract_base/django.py#L253-L287